src/components/SingleSelect/SingleSelect.less

```diff
@@ -0,0 +1,68 @@
+.lucid-SingleSelect {
+
+	.lucid-SingleSelect-Control {
+		.no-safari-flicker();
+		.box-sizing();
+		.gradient-animation( @featured-color-default-gradientStartColor, @featured-color-default-gradientEndColor);
+		cursor: pointer;
+		height: 28px;
+		display: flex;
+		outline: none;
+		border: 1px solid @featured-color-default-borderColor;
+		border-radius: @Button-size-borderRadius;
+		font: @font;
+		color: @color-black;
+		letter-spacing: .05em;
```

Comment:
I'm feeling like `em` and `rem` don't make sense in our library since that means our consumers could have wonky that aren't what we intended them to look like.